src/Rover.cpp

```diff
@@ -131,7 +132,9 @@ int main(int argc, char** argv) {
 	parseCommandLine(argc, argv);
 	Globals::AUTONOMOUS = false;
 	Globals::websocketServer.start();
-	auto mcProto = std::make_unique<net::mc::MissionControlProtocol>(Globals::websocketServer);
+	autonomous::AutonomousTask autonomous_task;
+	auto mcProto = std::make_unique<net::mc::MissionControlProtocol>(Globals::websocketServer,
```

Comment:
As we discussed, the autonomous task instantiation should happen in `MissionControlProtocol`.